packages/teleport-component-generator-html/src/plain-html-mapping.ts

```diff
@@ -27,7 +27,7 @@ export const PlainHTMLMapping: Mapping = {
       elementType: 'lottie-player',
       dependency: {
         type: 'package',
-        path: 'lottie-player',
+        path: '@lottiefiles/lottie-player',
```

Comment:
But @elizamaria this shouldn't effect anything right 🤔. Because `html-generators` basically takes `importAlias` and creates `script` tags ?? Was it creating any issues ?

Reply:
Yes, it was breaking when exporting the project in codesandbox.

Reply:
Thanks a lot for the fix, JK